### ISSUE TITLE
Rviz plugin requires kinematics.yaml to be loaded into it's parameter namespace?

### ISSUE THREAD

**Issue by mikeferguson** [moveit/moveit_ros#294]
I just spent a considerable amount of time tracking down why my interactive markers would not show up anymore, only to realize my problem was that I loaded rviz using "rosrun rviz rviz" and not using the rviz launch file from my moveit config.

The only error message that pops out of rviz is: 

```
[ INFO] [1376184839.868664941]: No active joints or end effectors found
for group 'arm'. Make sure you have defined an end effector in your SRDF
file and that kinematics.yaml is loaded in this node's namespace.
```

It wasn't until I turned on DEBUG for the ros.roscpp logger that I saw the parameter loading failing (and, by the way, rviz is spamming the crap out of this logger, so it isn't quickly noticeable).

TLDR: we should probably add a fallback whereby if there are no kinematics modules loaded into our namespace, we check the move_group namespace. Is there any reason not do this?

**Comment by isucan** [comment]
This is a duplicate of #180

**Comment by davetcoleman** [comment]
heh i've had this issue many times. i'm the one who added the info message "Make sure you have defined an end effector in your SRDF file and that kinematics.yaml is loaded in this node's namespace." :)

**Comment by mikeferguson** [comment]
I guess the "this node" didn't correspond to "RVIZ" in my head...

**Comment by isucan** [comment]
Believe it or not, I have had this same issue several times. It is relatively easy to fix (does my suggestion in #180 make sense?). I will do it this week.
